scss/_navbar.scss

```diff
@@ -77,9 +77,8 @@
   }
 
   .dropdown-menu {
-    position: static !important;
+    position: static;
```

Comment:
What if we scoped this to a `max-width` media query instead? Would that mean we don't need to reset the styles later for `.dropdown-menu` and `.dropdown-menu-right`?